functions/function/http-request-function/src/main/java/org/springframework/cloud/fn/http/request/HttpRequestFunctionConfiguration.java

```diff
@@ -24,11 +24,13 @@
 
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
```

Comment:
**[NIT]** copyright ⬆️

Reply:
Yeah…  In Spring Integration I have Gradle task which updates it for me automatically for all the modified files when I compile project. So, bad habit to forgot 😅.

on the other hand: no claims that this is going to be merged in 2022. So, I’ll just wait for the final decision  😉

Reply:
> no claims that this is going to be merged in 2022
Bahaha. Fair enough.